src/main/java/be/uantwerpen/minelabs/block/Blocks.java

```diff
@@ -144,6 +150,19 @@ private static <T extends Block> T register(T block, String identifier) {
         return Registry.register(Registry.BLOCK, new Identifier(Minelabs.MOD_ID, identifier), block);
     }
 
+    private static FluidBlock register(FlowableFluid fluid, Molecule molecule, String identifier) {
```

Comment:
I'm sorry for only spotting this now. Could you extract this into a class? This should make it easier to extend and more clear where the functionality is.